src/lib/components/sections/footer.svelte

```diff
@@ -1,257 +1,180 @@
 <script lang="ts">
-  import { onMount } from 'svelte';
-  import Instagram from '$lib/components/icons/instagram.svelte';
-  import Discord from '$lib/components/icons/discord.svelte';
-  import Youtube from '$lib/components/icons/youtube.svelte';
-  import Linkedin from '$lib/components/icons/linkedin.svelte';
-  import AcmToggle from '$lib/components/utils/acm-toggle.svelte';
-  import { AcmTheme, theme } from '$lib/stores/theme';
-
-  let jsEnabled = false;
-  onMount(() => (jsEnabled = true));
+  import Instagram from '../icons/instagram.svelte';
+  import Discord from '../icons/discord.svelte';
+  import YouTube from '../icons/youtube.svelte';
+  import LinkedIn from '../icons/linkedin.svelte';
```

Comment:
For the sake of consistency, I think we should import the icons from our internal `/$lib/` folder.
```suggestion
  import Instagram from '$lib/components/icons/instagram.svelte';
  import Discord from '$lib/components/icons/discord.svelte';
  import YouTube from '$lib/components/icons/youtube.svelte';
  import LinkedIn from '$lib/components/icons/linkedin.svelte';
```

Reply:
Resolved by <https://github.com/EthanThatOneKid/acmcsuf.com/pull/401/commits/084501e0c46803937cc23544bf3c502ff10d47a2>.